operations/workplace/people/README.md

```diff
@@ -1,15 +1,15 @@
 ---
-description: >-
+Description: >-
   Welcome to the Mattermost People Team Handbook Page. This page is being
   updated and actively worked on. If you have any question that you don't see
-  answered here, see PeopleOps on Mattermost.
+  answered here, see People Ops on Mattermost.
```

Comment:
@natalie-hub, I tried to add a link to the channel but it messed up the formatting. Should we add "channel" to this to clarify?